### ISSUE TITLE
[BUG] aria-label on kirby-radio-group is not forwarded to ion-radio-group

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#3795]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
Setting an aria-label on kirby-radio-group has no effect, as the form-control is actually the ion-radio-group within. We should take any aria-label attribute added to kirby-radio-group and set it on ion-radio-group instead. 

Please note - we already have support for correct labelling when the radio group is used inside a form-field, so this is only for 'standalone' radio groups with aria-label attributes added. 

## Describe how to reproduce the bug

1. Go to https://cookbook.kirby.design/branch/kby-develop/#/examples/radio
2. Inspect the default radio-group and add an `aria-label` to the `kirby-radio-group` element
3. Scan the page with e.g. level access extension
4. See that the control is still missing an accessible name.

The proposed solution can be further validated by adding the label to `ion-radio-group` instead. Scanning again, the missing label error is now gone.

## Which Kirby version was used?
10.3.0

## Please complete the following information:
- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]
- Browser version [e.g. 22]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
